parametric_plasma_source/plasma.py

```diff
@@ -0,0 +1,380 @@
+
+import os
+from pathlib import Path
+import tempfile
+import shutil
+from .source import source_sampling_cpp, plasma_source_cpp, plasma_source_hpp, make_file
+
+
+class Plasma():
+
+    def __init__(self,
+                 elongation=2.,
```

Comment:
I think these defaults should probably go away, is there anything meaningful in these defaults?

Reply:
Yep the defaults can go away. I was half thinking ITER defaults might be nice